impedance/models/circuits/elements.py

```diff
@@ -32,7 +22,14 @@ def wrapper(p, f):
         wrapper.__name__ = func.__name__
         wrapper.__doc__ = func.__doc__
 
+        global circuit_elements
+        if func.__name__ not in circuit_elements:
```

Comment:
I noticed as I was playing around that once I ran the code to define an element inside a Jupyter notebook, editing and rerunning the code didn't overwrite the element function because this line would only add it to circuit elements the first time. I'm wondering if it'd make sense to you to remove the if statement which would allow overwriting the base elements (maybe not ideal), but would also allow iterating in a notebook more easily?